src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -394,3 +394,64 @@ function rrule(
     end
     return Ω, lyap_pullback
 end
+
+#####
+##### `kron`
+#####
+
+@static if VERSION ≥ v"1.9.0"
```

Comment:
Was the AD problem introduced by https://github.com/JuliaLang/julia/pull/43724? Then this check could be improved to
```suggestion
@static if VERSION ≥ v"1.9.0-DEV.1267"
```

Reply:
I don't know if that's the case.

Reply:
It looks so, given that that PR added `kron!`. @devmotion I've always wondered, how does someone find out the Julia dev version for a particular commit?

Reply:
You can obtain it with the contrib/commit-name.sh script in the Julia repo: https://github.com/JuliaLang/Compat.jl#developer-tips

Reply:
I can't check it myself, but if it's obvious to you, I can approve this suggestion